ui/js/directives/treeherder/clonejobs.js

```diff
@@ -229,14 +229,9 @@ treeherder.directive('thCloneJobs', [
 
                 job = jgObj.jobs[l];
 
-
-                job.searchStr = thJobSearchStr(job) + ' ' + job.ref_data_name  + ' ' +
-                    job.signature;
-
                 // Keep track of visibility with this property. This
                 // way down stream job consumers don't need to repeatedly
                 // call showJob
```

Comment:
You should remove the comment above, since it no longer applies. Maybe put it in the new `addAdditionalJobParameters` function?